patients/consumer.go

```diff
@@ -219,6 +316,52 @@ func (p *PatientCDCConsumer) getUserEmail(userId string) (string, error) {
 	return user.Username, nil
 }
 
+func (p *PatientCDCConsumer) getUserRestrictedTokens(userId string) (clients.RestrictedTokens, error) {
+	p.logger.Debugw("Fetching restricted tokens by user id", "userId", userId)
+
+	restrictedTokens, err := p.auth.ListUserRestrictedTokens(userId, p.shoreline.TokenProvide())
+	if err != nil {
+		if e, ok := err.(*status.StatusError); ok && e.Code == http.StatusNotFound {
```

Comment:
This can never be true, because the auth client will return `nil, nil` if `auth` returns `404`.